composer.json

```diff
@@ -19,15 +19,15 @@
     "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ^2.6.6 || ~3.0",
-        "symfony/doctrine-bridge": "~2.3 || ~3.0",
+        "symfony/framework-bundle": "~2.7 || ~3.0",
+        "symfony/doctrine-bridge": "~2.7 || ~3.0",
```

Comment:
because it was too much hassle. this will go into the 1.4 version of the
bundle, and we only just released 1.3 today, so we expect symfony 2.3 to
be EOL when this gets released